packages/coclobas/coclobas.0.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Seb Mondet <seb@mondet.org>"
+authors: "Seb Mondet <seb@mondet.org>"
+homepage: "https://github.com/hammerlab/coclobas"
+bug-reports: "https://github.com/hammerlab/coclobas/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/hammerlab/coclobas.git"
+build: [
+  [make "byte"]
+  [make "native"]
+  [make "META"]
+  [make "coclobas.install"]
+]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "solvuu-build" {build & >= "0.3.0"}
+  "nonstd"
+  "sosa"
+  "pvem_lwt_unix"
+  "cohttp"
+  "lwt"
```

Comment:
I added constrains (there was already one on `lwt` through the `pvem_lwt_unix` package BTW).

The problem was only in the “`2.0`” builds (as in opam 2 I guess?); could this be a deeper issue?